react-jsx/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "react-jsx",
+  "version": "1.0.0",
+  "description": "",
+  "main": "webpack.config.js",
+  "scripts": {
+    "build": "webpack",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "@babel/core": "^7.18.13",
+    "@babel/plugin-transform-arrow-functions": "^7.18.6",
+    "@babel/plugin-transform-block-scoping": "^7.18.9",
```

Comment:
For general purposes and specifically for React we don't need these 2 plugins. We only need the `transform-react-jsx` plugin.